docs/src/pages/guides/notifications.md

```diff
@@ -104,12 +104,19 @@ To use Gmail SMTP with Monika,
 2. Use `smtp.gmail.com` for `hostname`.
 3. Use `587` for `port`.
 4. Use your Gmail address for `username`.
-5. Use your Gmail password for `password`. If you have activated 2-Factor-Authentication (2FA), you need to [create an App Password from your Account Settings](https://myaccount.google.com/security). Then use the app password for `password`.
+5. Use your Gmail password for `password`.
+   1. If you have activated 2-Factor-Authentication (2FA), you need to <a className="text-blue-700 font-semibold" href="https://support.google.com/accounts/answer/185833">create an App Password from your Account Settings</a>. Then use the app password for `password`.
```

Comment:
don't use the styling for link. we will change the styles later when we revamp the landing page and doc